codelists/models.py

```diff
@@ -392,7 +392,7 @@ class Meta:
 
     @property
     def term_or_code(self):
-        return self.term or self.code
+        return self.term or f"code: {self.code}"
```

Comment:
Is there supposed to be a space after the colon here?

Reply:
Yes, this is for display.